<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowFigWidget</class>
 <widget class="QWidget" name="ShowFigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
     border-width: 1px;
     border-style: solid;
     border-color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>119</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_draw">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>119</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>涂鸦</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_send">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>119</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>119</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
